daemon/kmd/wallet/driver/ledger_hid.go

```diff
@@ -82,6 +83,13 @@ func (l *LedgerUSB) WritePackets(msg []byte) error {
 		if err != nil {
 			return err
 		}
+		// on Windows:
+		// The usb library adds one extra byte to the input passed to the USB device
+		// so the written bytes are larger than what we've send
+		// https://github.com/karalabe/hid/blob/9c14560f9ee858c43f40b5cd01392b167aacf4e8/hid_enabled.go#L167
+		if runtime.GOOS == "windows" && cc > 0 {
```

Comment:
consider refactoring so you would use ledger_windows.go for windows-specific logic.